Source/WebKit/UIProcess/ProvisionalFrameProxy.cpp

```diff
@@ -87,7 +87,7 @@ ProvisionalFrameProxy::ProvisionalFrameProxy(WebFrameProxy& frame, Ref<WebProces
     // FIXME: Add more parameters as appropriate.
     // FIXME: This gives too much cookie access. This should be removed after putting the entire frame tree in all web processes.
     auto giveAllCookieAccess = LoadedWebArchive::Yes;
-    if (newProcess) {
+    if (shouldUseNewProcess) {
         page.websiteDataStore().networkProcess().sendWithAsyncReply(Messages::NetworkProcess::AddAllowedFirstPartyForCookies(m_process->coreProcessIdentifier(), WebCore::RegistrableDomain(request.url()), giveAllCookieAccess), [process = m_process, loadParameters = WTFMove(loadParameters), pageID = m_pageID] () mutable {
```

Comment:
Why the blank line here?

Reply:
What blank line?

Reply:
I don't see one in the source. I guess I got confused by the way Github presented the diff. It looked like there was a blank line after the if check.